lib/testsuite/selector-playground/index.js

```diff
@@ -0,0 +1,96 @@
+const vm = require('vm');
+const {crxfile} = require('nightwatch-selector-playground');
+const Debuggability = require('../../utils/debuggability');
+const Utils = require('../../utils');
+const WebSocket = require('./playground-listener');
+const {Logger} = Utils;
+
+module.exports = class SelectorPlaygroundServer extends WebSocket {
+  constructor() {
+    super();
+    this.initSocket(this.playgroundWrapper.bind(this));
+  }
+
+  setClient(client) {
+    this.client = client;
+  }
```

Comment:
In order to accomplish that, it is necessary for me to have a `client` instance at the time of creating the `playgroundServer`. Because I've initialised it in `runnerSetup` and till the time we don't create the `client` instance. That's why I have created a setter.

Reply:
if we move the logic to createSession at testsuite level we would have access to `client` then.

Reply:
moved 👍